webap/static/seechange.js

```diff
@@ -278,9 +309,9 @@ seechange.Exposure.prototype.render_page = function()
 
     var h2, h3, ul, li, table, tr, td, th, hbox, p, span, tiptext, ttspan;
 
-    rkWebUtil.elemaker( "p", this.div, { "text": "[Back to exposure list]",
-                                         "classes": [ "link" ],
-                                         "click": () => { self.exposurelist.render_page(); } } );
+    // rkWebUtil.elemaker( "p", this.div, { "text": "[Back to exposure list]",
+    //                                      "classes": [ "link" ],
+    //                                      "click": () => { self.exposurelist.render_page(); } } );
```

Comment:
leave or kill?

Reply:
Survive for now.

Internal navigation in webaps is always a bit of an issue, because the browser's back button often doesn't do what you want.  (It doesn't go to the preview view you had in your webap, it leaves your webap altogether.)  I've seen ways to work around this, but all of them are ugly.